app/views/home/index.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="container">
+  <h1>This is our events page</h1>
+
+  <% if current_user %>
+    <%= link_to events_path do %>
+    <button class="border-0 bg-info text-light px-3 py-2 rounded mr-2">Show my events</button>
+    <% end %>
+
+    <%= link_to current_user do %>
+    <button class="border-0 bg-info text-light px-3 py-2 rounded mr-2">Events i am attending</button>
+    <% end %>
+
+
+    <h1 class="mt-5">All Events</h1>
+    <div class="d-flex flex-wrap">
+      <%= render partial: 'event_card' %>
+    </div>
+  <% end %>
```

Comment:
- [ ] use a helper method to render the if/end statement in this part of your code there should be no if/end statements within views if you have several lines you can use a partial to render the markup